kornia/geometry/warp/homography_warper.py

```diff
@@ -292,12 +292,13 @@ def normal_transform_pixel(height: int, width: int) -> torch.Tensor:
                            [0.0, 1.0, -1.0],
                            [0.0, 0.0, 1.0]])  # 3x3
 
-    tr_mat[0, 0] = tr_mat[0, 0] * 2.0 / (width - 1.0)
-    tr_mat[1, 1] = tr_mat[1, 1] * 2.0 / (height - 1.0)
+    # prevent divide by zero bugs
+    width_denom: float = 1e-14 if width == 1 else width - 1.0
+    height_denom: float = 1e-14 if height == 1 else height - 1.0
```

Comment:
I would add `eps` to the function signature so that can be easier to tweak the function. Besides, something that can be helpful at testing time is to add a value check in the height/width and raise and error for < 1

Reply:
Sure, that's updated in 9cabc83